docs/tutorials/custom-vnet.md

```diff
@@ -96,19 +96,22 @@ In this case, we are going to use the following template:
   "apiVersion": "vlabs",
   "properties": {
     "orchestratorProfile": {
-      "orchestratorType": "Swarm"
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "networkPlugin": "kubenet"
+      }
     },
     "masterProfile": {
-      "count": 3,
-      "dnsPrefix": "",
+      "count": 1,
+      "dnsPrefix": "test",
```

Comment:
let's leave the dns prefix value empty since it's a unique value per cluster

Reply:
done